include/xkbcommon/xkbcommon.h

```diff
@@ -1144,11 +1144,36 @@ xkb_keymap_mod_get_name(struct xkb_keymap *keymap, xkb_mod_index_t idx);
 /**
  * Get the index of a modifier by name.
  *
+ * The [virtual modifiers] mappings to [real modifiers] is an implementation
```

Comment:
Maybe put it under a little heading?

Or maybe it will be even better in some separate FAQ docs page..

Reply:
I like the FAQ page idea. It was in my mind for some time, but I did not wanted to start it with a single question. Today I took the time to start such a proper page.